### DISCUSSION TITLE
카카오지도 문의요

### DISCUSSION THREAD

**Discussion by ranware** [changdoc/naver-vaccine-macro#209] [Q&A]
설명 해주신대로 했는데요 

카카오 지도랑 네이버 지도랑 같이 켜둬야 되나요?

아니면 카카오지도에서 실행만 하면 되나요?

**Comment by 1q2w3e4r-1** [comment]
저는 카카오지도만 켜놨습니다
잔여 나오니까 카카오켜놨던 페이지에서 네이버예약 페이지로 들어가지더라고요

**Reply by ranware** [reply]
아하 감사합니다
